samples/sample_common/include/version.h

```diff
@@ -30,7 +30,7 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #endif
 
 #ifndef MSDK_RELEASE
-#define MSDK_TARGETAPIMINOR 27
+#define MSDK_TARGETAPIMINOR 33
```

Comment:
It seems we don't need this line at all if you drop this line below:

`ss << MSDK_MAJOR << "." << MSDK_MINOR << "." << MSDK_TARGETAPIMINOR << "." << MSDK_BUILD;`

And why are we doing this each time in the release? Can we do that in master permanently?

Reply:
Will do.